### PR TITLE
Move the documentation to be more visible

### PR DESCRIPTION
**Pull request by tverbele**
I have moved the documentation markdown files in order to be more easily findable in the src tree:
* Moved them from distribution/src/markdown/docs to /docs in project root
* Added a README.md to point to the markdown files since github has nice markdown viewer
* Adapted the distribution build script to look at the new location for docs

Once merged we can also update the website to directly source the documentation pages from the github

### CODE REVIEW

**Comment by tverbele** [comment]
This also addresses issue #22

File: docs/contributor/build-concierge.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ This will clean workspace, build and run tests, and will create a distribution f
 
 ## Versioning
 
-The framework and the bundles jars will be named to Bundle-SymbolicName and Bundle-Version, e.g. `org.eclipse.concierge-5.1.0.SNAPSHOT-20151114134106.jar`. The Manifest.MF will specify the `Bundle-Version: 5.1.0.qualifier`, qualifier will be replaced by the current build date/time (MANIFEST driven approach). The version maintenance is up to the individual bundles/framework dependent on compatibility of changes.
+The framework and the bundles jars will be named to Bundle-SymbolicName and Bundle-Version, e.g. `org.eclipse.concierge-0.9.0.20141230122000.jar`. The Manifest.MF will specify the `Bundle-Version: 0.9.0.qualifier`, qualifier will be replaced by the current build date/time (MANIFEST driven approach). The version maintenance is up to the individual bundles/framework dependent on compatibility of changes.
```
Comment (JochenHiller) [inline docs/contributor/build-concierge.md]: the sample version should be better `5.1.0.SNAPSHOT-<current-timestamp>.jar`
Also the Bundle-Version should be `5.1.0.qualifier`

File: docs/contributor/build-concierge.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Concierge will be build using [Gradle](http://gradle.org/). The gradle wrapper s
 ```gradle
 // task to create a gradle wrapper with used gradle version
 task wrapper(type: Wrapper) {
-    gradleVersion = '2.8'
+    gradleVersion = '2.2.1'
     jarFile = '.gradle-wrapper/gradle-wrapper.jar'
```
Comment (JochenHiller) [inline docs/contributor/build-concierge.md]: I think 2.2.1 is outdated, we use now 2.8

File: docs/contributor/build-concierge.md [hunk 3]
```diff
@@ -28,9 +28,9 @@ This will clean workspace, build and run tests, and will create a distribution f
...
 
-The version for the distribution will be defined in `version.txt` in root directory. The used release plugin will automatically read in this file and use it as `project.version` property. When this file will have a `.SNAPSHOT` ending, it will be considered as a SNAPSHOT version. When doing a release (by calling `./gradlew release`) it will remove SNAPSHOT, make a release build, create a tag in git, and increase the version to next number.
+The version for the distribution will be defined in `version.txt` in root directory. The used release plugin will automatically read in this file and use it as `project.version` property. When this file will have a `-SNAPSHOT` ending, it will be considered as a SNAPSHOT version. When doing a release (by calling `./gradlew release`) it will remove SNAPSHOT, make a release build, create a tag in git, and increase the version to next number.
```
Comment (JochenHiller) [inline docs/contributor/build-concierge.md]: I think `-SNAPSHOT` is correct, and not `.SNAPSHOT`

File: docs/concierge-embedding.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ Once you have created and started the Framework instance, you can interact with
 
 ```
 BundleContext context = concierge.getBundleContext();
-Bundle shell = context.installBundle("file:org.eclipse.concierge.shell-5.0.0.jar"));
+Bundle shell = context.installBundle("file:org.eclipse.concierge.shell-0.9.0.jar"));
 shell.start();
```
Comment (JochenHiller) [inline docs/concierge-embedding.md]: 5.0.0 is the current version to be used

File: docs/concierge-commandline.md [hunk 5]
```diff
@@ -85,8 +72,11 @@ time, you can set the `org.osgi.framework.storage.clean` property to `onFirstIni
...
+The `repo` property is declared pointing to the Apache Felix repository, and is used later
+in the commands as `${repo}`.
+
+Finally all bundles are installed and started with the `-istart` command. This can take 
+both a web URL or a file system URL.
+
 
-Finally all bundles are installed and started with the `-istart` command or using `-install` and `-start` command. This can take both a web URL or a file system URL.
 
-If you specify a `-initlevel <level>` all bundles installed or started later will start with the given start level.
```
Comment (JochenHiller) [inline docs/concierge-commandline.md]: I think the docs you used have been outdated docs from Website. We should use the version from the old "docs" folder

File: docs/concierge-bundles.md [hunk 6]
```diff
@@ -23,6 +22,3 @@ Provides an implementation of the `StartLevel` service.
 
 Allows any JAXP compliant XML Parser to register itself as an OSGi parser service.
 
-### org.eclipse.concierge.service.eventadmin
-
-Provides a very lightweight implementation of EventAdmin (just 3 clasees).
```
Comment (JochenHiller) [inline docs/concierge-bundles.md]: I think the docs you used have been outdated docs from Website. We should use the version from the old "docs" folder

**Comment by JochenHiller** [comment]
Thanks for this contribution.
Good to have a README.md file as project main page at Github, will solve #22 
I also agree to move the docs to top-level folder, as it makes much easier to find and contribute to that.
But: I think you did not move the latest versions from distribution/src/markdown/*, but from webpage. It seems that most changes files are outdated compared to latest version. Except bndools, which seems to be more uptodate.
When we will merge that, I propose to squash all changes into one single commit.

**Comment by tverbele** [comment]
You are right Jochen, I have changed the docs to the latest versions from the distribution/src/markdown folder. Can you check again?

**Comment by JochenHiller** [comment]
Looks good now
I will merge and squash

**Pull request merged**